standards/qa-process.md

```diff
@@ -0,0 +1,132 @@
+# QA Sprint Process
+## Project-Under-Test is Prepared
+- For mobile apps: Application is made available on Testflight or loaded onto test devices. Application should be deployed to the Client’s Apple and Google accounts.
+- For web apps: Site is deployed to a staging server. Staging server should live on clients hosting provider.
+- Developer or PM prepares an Acceptance Test per the Shift3 guidelines.
+- PM or Lead Dev will create a milestone for the QA Sprint with:
+  - Milestone deadline
+  - Short project or feature description
+- PM or Lead Dev will create a project board for the QA Sprint.
+  - Pending
+  - Accepted
+  - In Progress
+  - Ready for Retest
+  - Done
+  - Rejected
+  
+  ![project board](./assets/qa-process/4.png)
+
+- PM or Lead Dev will create the “QA Sprint” issue with:
+  - Short project or feature description
+  - Delivery/testing timeframe
+  - Associated milestone
+  - Associated project board
+  - Sandbox URL/ Testflight App Name
+  - App Version number(s) to be tested
+  - Devices / Browsers to be tested
+  - Definition of Done
+  - Acceptance Test in Markdown Format.
+
+  ![qa sprint issue](./assets/qa-process/2.png)
+
+## Project Assigned
+1. PM or Lead Dev reaches out to Corey to schedule QA 
+a. PM sends an email to qa@shift3tech.com. Answer the following questions:  
+ **i.** Project and Client Name
+ **ii.** Task description  
+ **iii.** Due Date  
+ **iv.** Names of PM and Lead Dev  
+2. Corey receives the project and assigns it to the QA team / PM on monday.com. Deadline is set up, a timeline is scheduled, and calendar invites are sent.
+
+## Testing Kickoff
+1. Testing kickoff meeting occurs. PM or Dev will introduce the QA team to the Project-Under-Test.
+2. QA team meets to review the application and its purpose in order to better understand the most important functions of the website before diving in.
+3. PM / Dev defines the priorities for testing and defines the “Definition of Done”.
+4. Points of contact are defined.
+5. Identify known issues to be ignored or are outside of scope.
+
+## Access Given (during kickoff meeting)
+1. Desired browsers and devices to be tested on are identified.
+2. Team is assigned to the project repo on Github in order to post issues and coordinate with developers.
+3. If applicable, review for each user types abilities and limitations.
+4. Credentials are given to the team.
+
+# Testing Process
+## For QA:
+
+1. QA Testers will create a “Working” issue to work from, copy the Acceptance Test from the “QA Sprint” issue into the first comment, and use that to check off each tested feature.
+2. Discussions should occur constantly within the “Working” issue, tagging people as necessary.
+
+  ![github comments](./assets/qa-process/3.png)
+
+3. QA can generate new issues for discovered problems.
```

Comment:
For this line here can we include something that tells the developers to ensure they have proper access or is that implied?

Reply:
In this doc I'd say it's implied, but that may be a good item to bring up within Github project setup. In the same vein we should also mention adding `PR Team` to projects early on.